wdio.conf.js

```diff
@@ -0,0 +1,80 @@
+exports.config = {
+
+    // Define which test specs should run. The pattern is relative to the directory
+    // from which `wdio` was called. Notice that, if you are calling `wdio` from an
+    // NPM script (see https://docs.npmjs.com/cli/run-script) then the current working
+    // directory is where your package.json resides, so `wdio` will be called from there.
+    specs: [
+        './tests/specs/e2e/**/*.js'
+    ],
+
+    // ============
+    // Capabilities
+    // ============
+    // Define your capabilities here. WebdriverIO can run multiple capabilties at the same
+    // time. Depending on the number of capabilities, WebdriverIO launches several test
+    // sessions. Within your capabilities you can overwrite the spec and exclude option in
+    // order to group specific specs to a specific capability.
+    //
+    // If you have trouble getting all important capabilities together, check out the
+    // Sauce Labs platform configurator - a great tool to configure your capabilities:
+    // https://docs.saucelabs.com/reference/platforms-configurator
+    capabilities: [{
+        browserName: 'chrome'
```

Comment:
Needs to be either PhantomJS/SauceLabs/BrowserStack on CI.
Need to have a different config file for local development.